storage/postgres/testdb/testdb.go

```diff
@@ -27,8 +29,6 @@ import (
 	"time"
 
 	"github.com/google/trillian/testonly"
-
-	_ "github.com/lib/pq" // pg driver
```

Comment:
Isn't the driver required?

Reply:
Not according to goimports :)  No specific PG code gets executed in this file, so it isn't needed here.